rednotebook/configuration.py

```diff
@@ -53,6 +53,7 @@ class Config(dict):
         'mainFrameY': None,
         'leftDividerPosition': 260,
         'rightDividerPosition': None,
+        'tagDisplayLimit': 20,
```

Comment:
I think all tags should be displayed by default, but we can probably simply use a high number like 1000 for this. A value of 0 should show no tags.

Reply:
Yeah, it is a more reasonable approach - done.